elastic/eui
# [EuiStepsHorizontal] Create `size` prop to generate smaller step circles for horizontal steps

Fixes #6850 

## Summary
Creates the new `size` prop for `EuiStepsHorizontal`. This prop accepts `s` and `m` (small and medium) as values and defaults to `m`. `s` will produce a smaller circle around each step. 

![Horizontal steps size toggle](https://github.com/elastic/eui/assets/40739624/09fceab4-664a-495b-8d14-d7fe118ab376)

## QA

- Visit the [Horizontal Steps page](https://eui.elastic.co/pr_6928/#/navigation/steps#horizontal-steps) in staging and use the button group to change the step size from medium (default) to small. 
   - There should be no changes to the text size, just the circle surrounding the step number / status


### General checklist
